### DISCUSSION TITLE
how can I get a converge at first step by using ANK solver?

### DISCUSSION THREAD

**Discussion by hntter** [mdolab/adflow#255] [Q&A]
Dear Researchers, 

I am a beginer to use adflow. My case is a simple ellipsoid to flight at a condition of mach=0.03, altiude=20000m. I have tried to use ANK solver to start the calculation but it always print **killed** after startup. I believed it may be caused by convergence problems. Then I tried to use multigrid by tuning MGcycle to '5w'. It can work but the process still be killed when it swich to ANK solver. After that I also use the explame files of wing_vol.cngs and bwb.cgns to try the same script. Wing_vol.cgns can be solved by ANK solver but bwb.cgns still shows the same problem that bing killed after startup. May I ask how could I check what problems cause this and what parameters can help to solve this?

Best wishes, 

Zhengyu Qu

**Comment by eirikurj** [comment]
Please have a look at both the [ADflow](https://mdolab-adflow.readthedocs-hosted.com/en/latest/?badge=latest) and [MACH](https://mdolab-mach-aero.readthedocs-hosted.com/en/latest/?badge=latest) documentations for solver configuration and reference scripts. If your flight condition and problem setup is correct (very low speed and very high altitude), you will need to update the solver parameters. Read the solver docs carefully in order to understand what parameters to change. Furthermore, for future reference, please post the full solver output.

**Comment by hntter** [comment]
It seems the calculation is sensitive to the grid number. For instance, if the Grid Parameters 'N' is larger than 60 for my case, the calculation cannot start with ANK solver and will be killed at the first step. Thus I tuned 'N' to 50 for my case and the calculation can go well to a residual of 1e-8. Can any one shows some explainations for this problem?
